### PR TITLE
refactor deprecated array/string accessor syntax {} to []

### PR DESCRIPTION
**Pull request by yespire**
For error @ PHP8
PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /idna-convert/src/UnicodeTranscoder.php on line 206

"jsmitty12/phpwhois" is affected.

### CODE REVIEW

**Comment by blakethepatton** [comment]
Was about to make this same exact PR. Thanks @yespire. 

@algo26-matthias have you had a chance to look this one over?

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,38 @@
 /composer.phar
 /composer.lock
 /vendor
+
```
Comment (glensc) [inline .gitignore:7]: unrelated change, should go to a separate pull request.

**Comment by algo26-matthias** [comment]
Thanks, @yespire
